eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -300,7 +300,7 @@ jobs:
   - ${{ else }}:
     - ${{ if eq(parameters.targetOS, 'osx') }}:
       - script: |
-          $(sourcesPath)/src/runtime/eng/install-native-dependencies.sh osx
+          $(sourcesPath)/src/runtime/eng/common/native/install-dependencies.sh osx
```

Comment:
```suggestion
          $(sourcesPath)/eng/common/native/install-dependencies.sh osx
```

Reply:
Is it pointing to https://github.com/dotnet/sdk/blob/main/eng/common/native/install-dependencies.sh or https://github.com/dotnet/dotnet/blob/main/eng/common/native/install-dependencies.sh? We will need to mark it executable first. The one in runtime is executable.

Reply:
The latter: dotnet/dotnet

Reply:
Updating the sdk one should be sufficient to get the VMR one updated.